lib/stellar_core_commander/process.rb

```diff
@@ -300,9 +302,17 @@ def close_ledger
       true
     end
 
-    Contract None => Any
-    def set_upgrades
-      response = server.get('/upgrades?mode=set&upgradetime=1970-01-01T00:00:00Z&maxtxsize=10000&protocolversion=9')
+    Contract Or[Num, String] => Any
+    def set_upgrades(protocolversion="latest")
+
+      if protocolversion.eql? "latest"
```

Comment:
why not just `==` instead of `.eql` ?

Reply:
No particular reason, they behave identically in this scenario.

Reply:
the recommended way is to use `==` as `.eql` should only be used when implementing `==`

Reply:
Ok, just updated it. Squashed the commit also, since the change is really small.